en/docs/learn/examples/rabbitmq-examples/request-response-rabbitmq.md

```diff
@@ -41,14 +42,12 @@ See the instructions on how to [build and run](#build-and-run) this example.
         name="OrderProcessingService"
         transports="rabbitmq"
         startOnLoad="true">
-       <description/>
        <target>
         <inSequence>
-          <call>
-              <endpoint>
-                <http uri-template="http://localhost:8280/orders"/>
-              </endpoint>
-          </call>
```

Comment:
Shouldn't this backend call there as per the diagram. Let;s add an API to mock the backend. remember to change the port. Mention that this API is added to mock the back end.